src/components/Tools/AdvisorCard.vue

```diff
@@ -53,7 +53,9 @@ export default defineComponent({
         ...this.onboardingData.minor
           .map(acronym => getAdvisor(acronym, 'minor', this.userInfo))
           .flat(),
-      ].slice(0, this.maxItems);
+      ];
+      if (this.maxItems !== undefined) return packages.slice(0, this.maxItems);
+      return packages;
```

Comment:
```suggestion
      return packages.slice(0, this.maxItems);
```
If `maxItems` is `undefined` it'll select all of the items.

Reply:
that's the intention

Reply:
Yeah I mean if you remove the check it'll have the same effect

Reply:
oh really? ok cool